### DISCUSSION TITLE
0.7.25 stopped working

### DISCUSSION THREAD

**Discussion by SunDancerGE** [schwabe/ics-openvpn#1380] [Q&A]
After updating to 0.7.25 user gets following error message:

> "an openvpn profile needs ca certificate or fingerprint"

Users on 0.7.23 don't have any issues.

**Comment by schwabe** [comment]
Yes. This is a known issue that is fixed with 0.7.26

**Reply by SunDancerGE** [reply]
So just waiting for google to process the update then? Thanks for the quick reply.

**Reply by schwabe** [reply]
The update is already live but at 10% rollout. If you opt into the beta you should get it immediately or you wait a day or two until I up the rollout to 100%.
